edx_proctoring/tests/test_api.py

```diff
@@ -2448,3 +2450,207 @@ def test_get_integration_specific_email(self):
 
         del test_backend.integration_specific_email
         assert get_integration_specific_email(test_backend) == DEFAULT_CONTACT_EMAIL
+
+    @ddt.data(
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.declined, True),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.started, True),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.submitted, True),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.error, True),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.rejected, True),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.verified, True),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.declined, True),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.started, True),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.submitted, True),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.error, True),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.rejected, True),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.verified, True),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.declined, False),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.started, False),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.submitted, False),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.error, False),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.rejected, False),
+        (ProctoredExamStudentAttemptStatus.verified, ProctoredExamStudentAttemptStatus.verified, False),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.declined, False),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.started, False),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.submitted, False),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.error, False),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.rejected, False),
+        (ProctoredExamStudentAttemptStatus.rejected, ProctoredExamStudentAttemptStatus.verified, False),
+    )
+    @ddt.unpack
+    def test_grade_certificate_release_with_multiple_attempts(self, first_attempt_status, second_attempt_status, update_in_order):
```

Comment:
I grouped all of the test cases together, let me know if this is difficult to read and if I should add more comments or separate out tests